pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiDateQuestionFieldOrder.java

```diff
@@ -6,10 +6,14 @@
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 import org.jdbi.v3.stringtemplate4.UseStringTemplateSqlLocator;
 
-@UseStringTemplateSqlLocator
+
 public interface JdbiDateQuestionFieldOrder extends SqlObject {
 
+    @UseStringTemplateSqlLocator
     @SqlUpdate("insert")
```

Comment:
Maybe I misunderstood it, but this one should be either the SQL query or the template name of the SQL query. If it is a template name, then using just **insert** isn't the best option. If it is a SQL query, then obviously something should be fixed.

Reply:
This is using a StringTemplate method named `insert` in `JdbiDateQuestionFieldOrder.sql.stg`. The `@UseStringTemplateSqlLocator` annotation may be used at the interface-level to apply it to all of the methods.